schema/proto3/arionmaster.proto

```diff
@@ -23,27 +23,35 @@ option java_package = "com.futurewei.arion.schema";
 import "common.proto";
 
 service ArionMasterService {
+
   rpc RequestGoalstates (HostRequest) returns (NeighborRulesResponse) {
   }
 
   rpc RequestGoalstateStream (stream HostRequest) returns (stream NeighborRulesResponse) {
   }
+
 }
 
 service Watch {
 
-  rpc Watch (stream ArionWingRequest) returns (stream NeighborRule) {
+  rpc Watch (stream ArionWingRequest) returns (stream ArionWingResponse) {
   }
+
 }
 
 message ArionWingRequest {
-  string vni = 1;
-  string ip = 2;
-  uint64 rev = 3;
-  string group = 4;
+  uint64 rev = 1;
+  string group = 2;
+  string map = 3;
```

Comment:
what is 'map' for? So: 
1. neighbor and sg sharing 1 grpc channel
2. and 2 WingRequests? 
3. 'map' is goal state type?